docs/node/signals.md

```diff
@@ -41,28 +41,35 @@ Signal handlers can be either synchronous or asynchronous, in this example, our
 > Note that this example is a simplification of the recommended way to handle Signals [below](#triggers) since the Workflow cannot be cancelled unless it awaits a [cancellable operation](/docs/node/cancellation-scopes).
 
 ```ts
-// implementation
 import { Blocked } from '../interfaces';
 
-let unblock: () => void;
-
-const signals = {
-  // Unblock main by resolving the awaited Promise
-  unblock(): void {
-    if (unblock !== undefined) {
-      unblock();
-    }
-  },
+export const unblockWithSignal: Blocked = () => {
+  let blocked = true;
+  let unblock: () => void;
+
+  return {
+    signals: {
+      // Unblock main by resolving the awaited Promise
```

Comment:
Unblock `execute()` by..

Reply:
I should have grepped for main, missed it. Thanks!